File: PYTHON/PART_1/Oops/Inheritance/Multilevel.py
class Grandfather:
    def Home(self):
        print("Grandfather Home")
class Father(Grandfather)  :
    def bike(self):
        print("Father House")
class Son(Father):
    def Book(self):
        print("son Book")
son =Son()
son.bike()
son.Book()
son.Home()                

File: JAVA/Oops/Inheritance/Multilevel.java
package Oops.Inheritance;

public class Multilevel {
    public static void main(String[] args) {
        Son2 son=new Son2();
        son.bike();
        son.House();
        son.car();

    }
}
class GrandFather{
    void car(){
        System.out.println("GrandFather is Have a BMW Car.");
       }
}
class Father2 extends GrandFather{
    void House(){
        System.out.println("Father is Have Own 2bhk House.");
     }
}
class Son2 extends  Father2{
    void bike(){
        System.out.println("Son is Have a bike.");
       }
}
